brainstormforce/astra
# AST-2883 - fix: need to change learndash courses section name

### Description

- Need to change learndash courses section name
- Ex. Single sfwd courses → Single courses
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2883
### Screenshots

https://d.pr/i/T4FLyC

### Types of changes


- bugfix: changed dynamic cpt label to a more meaningful name



### How has this been tested?

- Install LearnDash plugin.
- Visit customizer LearnDash section, check Courses, Topics, other CPT's.
- The name of the section should be meaningful

### Checklist:
